### PR TITLE
Add ?platform for /experiments query

### PR DESCRIPTION
**Pull request by DzikStar**

### CODE REVIEW

**Comment by splatterxl** [comment]
Can you add an enum for this

File: pages/topics/experiments.mdx [hunk 1]
```diff
@@ -433,9 +433,10 @@ A fingerprint will only be generated and returned if no authorization or fingerp
 
 ###### Query String Parameters
 
-| Field                   | Type    | Description                                               |
-| ----------------------- | ------- | --------------------------------------------------------- |
-| with_guild_experiments? | boolean | Whether to include guild experiments in the returned data |
+| Field                   | Type    | Description                                                           |
+| ----------------------- | ------- | --------------------------------------------------------------------- |
+| with_guild_experiments? | boolean | Whether to include guild experiments in the returned data             |
+| platform?               | string  | Includes experiments from provided platform enum in the returned data |
```
Comment (splatterxl) [inline pages/topics/experiments.mdx:439]:
```suggestion
| platform?               | string  | Limits the response to experiments targeting the selected platform |
```

**Comment by splatterxl** [comment]
> Also. Coming back to the comment where you corrected the line - I wanted to point out that after adding `?platform=DEVELOPER_PORTAL` it combines default response with platform response. 
Oh, didn't know that -- does it add an extra field or just inserts into the array?

> Another thing, maybe I'll add tomorrow. Using `platform` seems to require authorization to get request response.
Does this also accept a fingerprint header?

File: pages/topics/experiments.mdx (hunk 1 above)
Comment (dolfies) [inline pages/topics/experiments.mdx:439]:
```suggestion
| platform?               | string  | Supplements the response with experiments from the given [platform](#experiment-platform) |

###### Experiment Platform

| Value | Description |
| ------ | ------------ |
| DEVELOPER_PORTAL | The developer portal |
```

File: pages/topics/experiments.mdx [hunk 2]
```diff
@@ -433,9 +433,15 @@ A fingerprint will only be generated and returned if no authorization or fingerp
 
 ###### Query String Parameters
 
-| Field                   | Type    | Description                                               |
-| ----------------------- | ------- | --------------------------------------------------------- |
-| with_guild_experiments? | boolean | Whether to include guild experiments in the returned data |
+| Field                   | Type    | Description                                                           |
+| ----------------------- | ------- | --------------------------------------------------------------------- |
+| with_guild_experiments? | boolean | Whether to include guild experiments in the returned data             |
+| platform?               | string  | Supplements the response with experiments from the given [platform](#experiment-platform) |
```
Comment (splatterxl) [inline pages/topics/experiments.mdx:439]:
```suggestion
| platform?               | string  | Whether to also include experiments for the given [platform](#experiment-platform) |
```

It's still unclear to me what the param actually does... 

Maybe worth it to add a sentence outside the table to clarify in what situation it's sent and why the data isn't included by default?

**Review by splatterxl — CHANGES_REQUESTED** [review]

**Pull request merged**